terraform/modules/fhinfrastructurestack/fh-report-api-app.tf

```diff
@@ -151,6 +152,7 @@ resource "azurerm_storage_account" "storage_rep_api_logs" {
   public_network_access_enabled = local.public_network_access_enabled_storage
   account_replication_type = local.account_replication_type
   infrastructure_encryption_enabled = local.infrastructure_encryption_enabled
+  cross_tenant_replication_enabled = true
```

Comment:
as per discussion, this might need be false

Reply:
Updated